src/EnergyPlus/api/func.cc

```diff
@@ -58,7 +58,9 @@
 void initializeFunctionalAPI(EnergyPlusState state)
 {
     auto *thisState = reinterpret_cast<EnergyPlus::EnergyPlusData *>(state);
-    thisState->dataInputProcessing->inputProcessor = EnergyPlus::InputProcessor::factory();
+    if (!thisState->dataInputProcessing->inputProcessor) {
+        thisState->dataInputProcessing->inputProcessor = EnergyPlus::InputProcessor::factory();
+    }
```

Comment:
This was a brutal thing to find.  This was causing the input processor to be wiped out right in the middle of the get input operations.  So fun!  I definitely didn't lose a bunch of hours trying to figure out why...